src/pages/BrowseData.js

```diff
@@ -0,0 +1,277 @@
+    backgroundColor: 'white'
+  }
+}, [
+  img({
+    src, alt, height: '60%', width: 'auto'
+  })
+])
+
+
+const Participant = ({ logoBox, children }) => h(FadeBox, [
+  div({ style: { display: 'flex' } }, [
+    logoBox,
+    div({ style: { flex: 1, margin: '1rem 0 0 4rem' } }, [children])
+  ])
+])
+
+
+const ParticipantControls = ({ size, helpLink, children }) => {
+  return div({ style: { display: 'flex', alignItems: 'center' } }, [
+    div({ style: { width: 200, flex: 'none' } }, [size || '']),
+    children,
+    h(InfoBox, { style: { marginLeft: '1rem' }, position: 'bottom', size: 20 }, [
+      div(['Look for the Export to Saturn icon to export data from this provider. ']),
+      // not used yet:
+      helpLink && link({
+        href: helpLink,
+        target: '_blank'
+      }, ['Learn more.'])
+    ])
+  ])
+}
+
+
+const NIHCommonsButtons = div({ style: { width: 303 } }, [
+  buttonPrimary({
+    style: { margin: '0.25rem 0', width: 303 },
+    as: 'a',
+    href: 'https://dcp.bionimbus.org/',
+    target: '_blank'
+  }, ['Browse Data via Windmill']),
+  buttonPrimary({
+    style: { margin: '0.25rem 0', width: 303 },
+    as: 'a',
+    href: 'https://commons.ucsc-cgp-dev.org/boardwalk',
+    target: '_blank'
+  }, ['Browse Data via Boardwalk'])
+])
+
+
+const BrowseData = pure(() => {
+  return h(Fragment, [
+    h(TopBar, {}, []),
+    div({ style: styles.top.container }, [
+      div({ style: styles.top.title }, ['Saturn']),
+      div({ style: styles.top.summary }, ['Browse Data from the largest biomedical research agencies in the world.'])
+      /*
+       * Need real numbers
+       * div({ style: styles.top.details }, ['Over 2,300,000 participants']),
+       * div({ style: styles.top.details }, ['Over 257,000 cells'])
+       */
+    ]),
+    div({ style: styles.banner.container }, [
+      div({ style: styles.banner.textContainer }, [
+        div({ style: styles.banner.title }, ['The next generation of research:']),
+        div({ style: styles.banner.details }, ['Aggregate data from our participating partners into Saturn'])
+      ]),
+      h(Clickable, {
+        disabled: true,
+        tooltip: 'Documentation coming soon',
+        style: {
+          textTransform: 'uppercase', fontWeight: 500,
+          padding: '1rem 1.5rem',
+          border: '1px solid',
+          borderRadius: 5
+        },
+        hover: {
+          backgroundColor: Style.colors.primary,
+          borderColor: Style.colors.primary
+        }
+      }, ['Learn How'])
+    ]),
+    div({ style: styles.content.container }, [
+      div({ style: styles.content.header }, ['Participating Research Databases']),
+      h(Participant, { logoBox: logoBox({ src: nhsLogo, alt: 'Nurses\' Health Study logo' }) }, [
+        div({ style: styles.content.title }, ['Nurses\' Health Study']),
+        p(['The Nurses\' Health Study and Nurses\' Health Study II are among the largest investigations into the ' +
```

Comment:
If you put this into a `` ` `` string literal you should be able to have a line break that won't be rendered (due to default `white-space` style) and avoid escaping the quotes.